content/concepts/protocols.md

```diff
@@ -1,227 +1,167 @@
 ---
-title: Protocols
-weight: 3
+title: Introduction to libp2p Protocols
+menuTitle: Protocols
+weight: 1
 ---
 
-There are protocols everywhere you look when you're writing network applications, and libp2p is
-especially thick with them.
-
-The kind of protocols this article is concerned with are the ones built with libp2p itself,
-using the core libp2p abstractions like [transport](/concepts/transport), [peer identity](/concepts/peers#peer-id/), [addressing](/concepts/addressing/), and so on.
-
-Throughout this article, we'll call this kind of protocol that is built with libp2p
-a **libp2p protocol**, but you may also see them referred to as "wire protocols" or "application protocols".
-
-These are the protocols that define your application and provide its core functionality.
-
-This article will walk through some of the key [defining features of a libp2p protocol](#what-is-a-libp2p-protocol), give an overview of the [protocol negotiation process](#protocol-negotiation), and outline some of the [core libp2p protocols](#core-libp2p-protocols) that are included with libp2p and provide key functionality.
-
+This article is concerned with protocols that compose libp2p
+using core libp2p abstractions like the [transport](/concepts/transport), 
+[peer identity](/concepts/peers#peer-id/), [addressing](/concepts/addressing/) 
+abstractions to name a few. 
 
 ## What is a libp2p protocol?
 
-A libp2p protocol has these key features:
+A libp2p protocol is a network protocol that includes
+the key features mentioned below.
 
-#### Protocol Ids
+#### Protocol IDs
 
-libp2p protocols have unique string identifiers, which are used in the [protocol negotiation](#protocol-negotiation) process when connections are first opened.
+libp2p protocols have unique string identifiers, which are used in the 
+[protocol negotiation](#protocol-negotiation) process when connections are first opened.
 
-By convention, protocol ids have a path-like structure, with a version number as the final component:
+By convention, protocol IDs have a path-like structure, with a version number as the 
+final component:
```

Comment:
```suggestion
Historically, the IDs of many protocols in use by libp2p have a path-like structure, with a version number as the 
final component:
```

then after the protocol ID:
Experience has shown that semver doesn't make a lot of sense for wire protocols, so you might see newer protocols without a version number.